packages/components/src/components/dropdown/Dropdown.tsx

```diff
@@ -182,14 +132,72 @@ export class Dropdown<T = LabelValue> extends React.Component<
       tooltipCloseLabel,
       value,
       variant,
-      bindLabel,
       ...otherProps
     } = this.props;
 
+    const handleChange = (selectedOption, meta: ActionMeta<T>) => {
```

Comment:
Any reason you moved component methods inside the render method?

Reply:
Ok got it, Why just not disable the rules for these props, instead of changing the structure of the component?

By adding above the props concerned
// eslint-disable-next-line @typescript-eslint/no-unused-vars

Reply:
```
const {
   ...
      // eslint-disable-next-line @typescript-eslint/no-unused-vars
      isOptionDisabled,
      // eslint-disable-next-line @typescript-eslint/no-unused-vars
      isOptionSelected,
   ... }
   = this.props
```

etc...